### DISCUSSION TITLE
about quetion

### DISCUSSION THREAD

**Discussion by smlin0513** [BartoszCichecki/LenovoLegionToolkit#1327] [Q&A]
My computer always shows this frequency.
<img width="770" alt="Snipaste_2024-06-08_11-33-17" src="https://github.com/BartoszCichecki/LenovoLegionToolkit/assets/18237744/9d5c8af3-4056-4cc0-ae9f-9ec9aaeab31c">

**Comment by Ace-Radom** [comment]
Plz attach logs as the README says and upload it here. Without a log there's no way for us to figure out what's happening.

**Reply by BartoszCichecki** [reply]
Most likely they set "Maxium performance" in nVidia Control Panel - it's the most common reason why dGPU is stuck at highest clocks. It's definitely not an LLT issue.
